docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+this structure.
+
+
+
+Steering Committee and Leadership Roles
+---------------------------------------
+
+Organizational Principles
+`````````````````````````
+The steering committee steers. The goal of the steering committee is to help
+guide the direction of the project. Decisions in the steering committee will 
+focus on how to present project issues to the broader community in a clear way
+rather than making project decisions without community input. 
+
+Issues or pull requests that can be reviewed by any community member can be
+reviewed and approved by the steering committee or steering committee members.
+The steering committee can decide that an issue should be prioritized for wider
+communal discussion or that a pull request requires more discussion or reviews
+than standard before merging. The steering committee can also decide how a
+breaking change (something that changes existing user function calls or program
+outputs) will be presented to the developer and user base for discussion, before
+decisions are made.
+
+Steering committee decisions should strive for consensus and accept the
+majority’s decision. If voting is necessary, it can happen asynchronously and is
+defined as more than half of the members of the steering committee. If a member
+of the steering committee is unable to spend the time necessary to understand
+an issue & vote, they can recuse from a decision so that the number of committee
+members is reduced
+
+Openness is a priority:
+
+- Openness is critical to building trust with the broader community
+- Openness provides a mechanism for non-steering committee members to identify
+  and address steering committee blindspots
+- Openness provides a smoother transition to onboard future steering committee
+  members
+- If steering committee discussions (written or verbal) could include
+  community members without compromising efficiency, they should be invited. The
+  steering committee can schedule discussions without needing to ask about the
+  availability for people outside of the steering committee. If notes from
+  meetings can be openly shared without compromising personal privacy, they
+  should be.
+
+The current members of the tedana steering committee are:
+
+- Name 1 & link to github profile
+- Name 2 & link to github profile
+- Name 3 & link to github profile
+- Name 4 & link to github profile
+- Name 5 & link to github profile
+
+If you are interested in joining the steering committee, please let the current members know.
+
+Leadership roles
+````````````````
+We have identified key skills that help advance tedana development and
+volunteers who will be the point-people for each of these roles. One person
+can fill more than one role and more than one person can decide to share or
+split the responsabilities of a role. The steering committee will include
+people who also take on leadership roles, but a person can take on a leadership
+role without also volunteering to be on the steering committee.
```

Comment:
Ah I see. Should this section go before `steering committee`, then, since folks will more directly work with those in leadership roles, and they're also easier to transition into ?